### PR TITLE
Fix a test that fails in CI and cargo fmt

### PR DESCRIPTION
**Pull request by bryangarza**

### CODE REVIEW

**Comment by pnkfelix** [comment]
(i told bryan in privmsg that I didn't want all the `if PRINT_MSG { ppIROp(...); }` one-liners to be replaced with three-liners. They are addressing that in some manner.)

**Comment by bryangarza** [comment]
> (i told bryan in privmsg that I didn't want all the `if PRINT_MSG { ppIROp(...); }` one-liners to be replaced with three-liners. They are addressing that in some manner.)

I added `use_small_heuristics = "Max"` to the `rustfmt.toml`

edit: never mind, I didn't use `use_small_heuristics = "Max"` and instead wrote a helper function for the `if PRINT_MSG...` lines

File: krabcake/rs_hello/src/lib.rs [hunk 1]
```diff
@@ -823,23 +792,20 @@ pub extern "C" fn rs_shadow_qop(
         if_sb_event_queued_print(
             || msg!(b"rs_shadow_qop sb event \0"),
             || {
-                if PRINT_MSG { ppIROp(op); }
+                if PRINT_MSG {
```
Comment (pnkfelix) [inline krabcake/rs_hello/src/lib.rs]: @bryangarza , you missed restoring some of these

Comment (bryangarza) [reply krabcake/rs_hello/src/lib.rs]: even with Max heuristics, some of these were being broken up by `cargo fmt`, so I made a helper function instead.

File: .githooks/pre-commit [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+set -eu
+
+# https://deaddabe.fr/blog/2021/09/29/git-pre-commit-hook-for-rust-projects/
+# https://asciiflow.com
+
+cd krabcake/rs_hello
+
+if ! cargo fmt -- --check
+then
+    echo '┌─────────────────────────────────────────────────────────┐'
```
Comment (pnkfelix) [inline .githooks/pre-commit:12]: lol box characters FTW

**Review by pnkfelix — APPROVED** [review]

**Pull request merged**